docker-compose.yml

```diff
@@ -32,7 +34,17 @@ services:
     mem_limit: 4g
     shm_size: 1g
     ports:
-      - "${POSTGRES_EXPOSE:-127.0.0.1:5432}:5432"
+      - "${POSTGRES_EXPOSE:-0.0.0.0:5432}:5432"
```

Comment:
It's good ? I'm not sure about what you say to me recently about default exposure of postgres

Reply:
Thank you for explainations 👍 I Will change it